decisions/mnm_hermetics_decisions.txt

```diff
@@ -346,12 +346,13 @@ targeted_decisions = {
 		filter = self
 		ai_target_filter = self
 
-		ai_check_interval = 12
+		ai_check_interval = 180
 
 		potential = {
 
 			# Warcraft
 			is_untouchable_trigger = no
+			only_rulers = yes
```

Comment:
What is a pre-trigger doing in `potential`? It broke the decision.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/75379562-06a31300-58ef-11ea-93bd-8f285c4b37a7.png)

</details>

Reply:
fixed